checkseum.opam

```diff
@@ -23,7 +23,7 @@ build: [
 
 depends: [
   "ocaml"         {>= "4.03.0"}
-  "dune"          {build}
+  "dune"          {build & >= "1.7"}
```

Comment:
The minimum version is a bit suspicious here. Didn't we make variants experimental only in 1.9.2? `(using library_variants 0.1)` will not work in earlier versions.